released/packages/coq-flocq/coq-flocq.3.2.0+8.11/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "guillaume.melquiond@inria.fr"
+homepage: "http://flocq.gforge.inria.fr/"
+dev-repo: "git+https://gitlab.inria.fr/flocq/flocq.git"
+bug-reports: "https://gitlab.inria.fr/flocq/flocq/issues"
+license: "LGPL 3"
+build: [
+  ["./autogen.sh"]
+  ["./configure"]
+  ["./remake" "-j%{jobs}%"]
+]
+install: ["./remake" "install"]
+depends: [
+  "coq" {= "8.11.0"}
```

Comment:
```suggestion
  "coq" {= "8.11.0"}
  ("conf-g++" {build} | "conf-clang" {build})
```
As discussed in #593.

Reply:
I guess the conf-g++ / conf-clang is inteded to say that this requires a C compiler, but I am not aware that Flocq requires a C compiler. As far as I can tell Flocq is a pure Coq development (not even OCaml code).

Reply:
Here is what happens when running `configure` of Flocq on Cygwin without a C++ compiler:
```shell
$ ./configure
checking for g++... no
checking for c++... no
checking for gpp... no
checking for aCC... no
checking for CC... no
checking for cxx... no
checking for cc++... no
checking for cl.exe... no
checking for FCC... no
checking for KCC... no
checking for RCC... no
checking for xlC_r... no
checking for xlC... no
checking whether the C++ compiler works... no
configure: error: in `/home/palms/src/coq-community/flocq-3.2.0':
configure: error: C++ compiler cannot create executables
See `config.log' for more details
```

Reply:
Flocq also seems to depend on autoconf during build (`conf-autoconf` package in OPAM). All of these dependencies are due to using the `remake` build system rather than, say, `coq_makefile` or dune.

Reply:
To be more accurate, `conf-autoconf` is not needed when using the official tarball. You only need it when compiling directly from a git repository. So, while it can be added as a dependency, it should be conditioned by `{ dev & build }`.

Reply:
@silene thanks for the clarification, but the archive here is not the official tarball, so `conf-autoconf` is needed for this particular package definition. I'll remember to adjust only the `dev` Flocq version in the `extra-dev` repo with the autoconf dependency.

Reply:
Thinking about it, I think Flocq needs a C compiler because it uses remake to build and remake is built from C sources in the configure step. So it does need a C compiler in the end.